guides/source/form_helpers.md

```diff
@@ -867,15 +998,15 @@ We can mix and match these two concepts. One element of a hash might be an array
 <input name="person[addresses][][city]" type="text"/>
 ```
 
-This would result in `params[:person][:addresses]` being an array of hashes with keys `line1`, `line2`, and `city`.
+This would result in `params[:person][:addresses]` being an array of hashes. Each hash in the array will have the keys `line1`, `line2`, and `city`.
```

Comment:
It would be nice to show the results here as well.
I haven't checked if the output is correct here...
````suggestion
This would result in `params[:person][:addresses]` being an array of hashes. Each hash in the array will have the keys `line1`, `line2`, and `city`.

```ruby
{ 'person' =>
  { 'addresses' => [
     { 'line1' => '1000 Fifth Avenue',
       'line2' => '',
       'city' => 'New York'
     },
    { 'line1' => 'Calle de Ruiz de Alarcón',
       'line2' => '',
       'city' => 'Madrid'
     }
     ]
  }
}
```
````

Reply:
Updated